pkg/identities/service.go

```diff
@@ -76,6 +91,59 @@ func (s *Service) parseError(r *http.Response) *kClient.GenericError {
 	return gerr.Error
 }
 
+func (s *Service) cookiesToString(cookies []*http.Cookie) string {
+	var ret = make([]string, len(cookies))
+	for i, c := range cookies {
+		ret[i] = fmt.Sprintf("%s=%s", c.Name, c.Value)
+	}
+	return strings.Join(ret, "; ")
+}
```

Comment:
question: since cookies is a slice it could potentially be `nil`. Are the arguments passed to this function possibly nil?